internal/kibana/saved_object/create.go

```diff
@@ -0,0 +1,51 @@
+package saved_object
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+)
+
+func (r *Resource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	var model ksoModelV0
+
+	resp.Diagnostics.Append(req.Config.Get(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	err := model.UpdateModelWithObject()
+	if err != nil {
+		resp.Diagnostics.AddError("failed to update model from object", err.Error())
+		return
+	}
+
+	kibanaClient, err := r.client.GetKibanaClient()
+	if err != nil {
+		resp.Diagnostics.AddError("unable to get kibana client", err.Error())
+		return
+	}
+
+	result, err := kibanaClient.KibanaSavedObject.Import([]byte(model.Imported.ValueString()), false, model.SpaceID.ValueString())
```

Comment:
```suggestion
	result, err := kibanaClient.KibanaSavedObject.Import([]byte(model.Imported.ValueString()), true, model.SpaceID.ValueString())
```

Given the use case here, I wonder if we should just always overwrite existing objects. It would simplify the workflow for bringing existing objects into TF, especially in a resource where a proper `import` handler is kind of tricky given the API constraints.

Reply:
I wondered about that and initially had overwrite set to `true`. I agree it would simplify importing resources into TF to have it set to `true`.